lib/wrapper.js

```diff
@@ -26,6 +29,13 @@ class WebSocketServerWrapper extends WebSocketServerChannel {
 
 		// When a WebSocket connects to the server...
 		this.server.on("connection", (socket, req) => {
+			// Set up heartbeat ("pong" listener) on raw socket, if needed
+			if(this.options.heartbeatInterval > 0) {
+				this._pong = true;
+				socket.on("pong", function() {
+					this._pong = true;
```

Comment:
`this._pong = true;`

should be

`socket.socket._pong = true;`

since on next line socket is added to the wrapper and pong is called after that.

Reply:
`this` will be equal to the raw socket, so I think this is okay.